components/autocomplete/style.scss

```diff
@@ -34,3 +34,7 @@
 		@include button-style__hover;
 	}
 }
+
+.autocomplete-token[data-mce-selected] {
```

Comment:
cc @jasmussen for color choice and whether we should show the background at all times, not just on selection (similar to the code inline format).

Reply:
I think @karmatosed is looking at this now. My thoughts are these:

- We should definitely leverage, if we can, the inline boundaries for tokens as well. 
- Inline boundaries, like bold, italic, links, should feel "ephemeral" and disappear when you're not interacting with the boundary. Subtle colors and no borders make them feel ephemeral.
- The tokens should feel more substantial, and should probably _keep_ their boundaries even when you're not interacting with them. A subtle background plus border can help make them feel like "chips", which I think would make them feel more substantial, which seems a good thing.